assets/translations/home.js

```diff
@@ -5,8 +5,9 @@ export default {
 			description:
 				'Front-End Foxes are people who identify as women and who want to learn front-end technologies' +
 				' to make websites and mobile apps',
-			workshops: 'Workshops',
-			patreon: 'Become a Patron',
+			workshops: 'Vue Workshops',
+			meetups: 'Meetups',
```

Comment:
Is Meetups currently in use? 👀 
FYI: https://frontendfoxes.slack.com/archives/GC7FZPGMQ/p1588696755062500

Reply:
@jlooper Is it still in use?
@ryamakuchi `Meetups` button is still present so I think I will leave the translation message on this PR though.

Reply:
LGTM! 😸